sdk/communication/Azure.Communication.Identity/tests/CommunicationIdentityClient/CommunicationIdentityClientLiveTests.cs

```diff
@@ -87,6 +82,35 @@ void ValidateScopesIfNotSanitized()
             }
         }
 
+        [TestCase("chat", TestName = "GettingTokenWithChatScope")]
+        [TestCase("voip", TestName = "GettingTokenWithVoipScope")]
+        [TestCase("chat.join", TestName = "GettingTokenWithChatJoinScope")]
+        [TestCase("chat.join.limited", TestName = "GettingTokenWithChatJoinLimitedScope")]
+        [TestCase("voip.join", TestName = "GettingTokenWithVoipJoinScope")]
+        [TestCase("chat", "voip", TestName = "GettingTokenWithChatVoipScopes")]
+        [TestCase("chat", "chat.join", "chat.join.limited", TestName = "GettingTokenWithAllChatScopes")]
```

Comment:
just out of curiosity, would the user have unrestricted access to the chat in this scenario? :)